app/src/main/java/org/mozilla/focus/widget/DownloadListAdapter.java

```diff
@@ -113,7 +113,9 @@ public void onBindViewHolder(RecyclerView.ViewHolder viewHolder, int position) {
                 final int progress = (int) (100 * downloadInfo.getSizeSoFar() / downloadInfo.getSizeTotal());
                 holder.progressBar.setProgress(progress);
                 holder.progressBar.setVisibility(View.VISIBLE);
-                subtitle = progress + "%";
+                Double downloadedBytes = (downloadInfo.getSizeSoFar() / 1024 / 1024);
```

Comment:
I think this is good enough to merge, just an FYI if it's me I'll excerpt these two doubles before
`final int progress = (int) (100 * downloadInfo.getSizeSoFar() / downloadInfo.getSizeTotal());`
and change it into
`final int progress = (int) (100 * downloadedBytes / totalBytes);`
to make thinks clear.
Feel free to submit another pr to do this.

Reply:
@mTwTm  Yes I will update this code accordingly and send another pull request.